packages/xo-server/config.toml

```diff
@@ -49,6 +49,11 @@ maxTokenValidity = '0.5 year'
 # Delay for which backups listing on a remote is cached
 listingDebounce = '1 min'
 
+# Configuration for backupNg
+[backupNgOptions]
+# timeout in milliseconds
+timeout = 2e3
```

Comment:
Like other settings, use string format to better represent the duration (and use `_parseDuration`).